src/compiler/emitter.ts

```diff
@@ -180,7 +180,7 @@ namespace ts {
             // Setup and perform the transformation to retrieve declarations from the input files
             const nonJsFiles = filter(sourceFiles, isSourceFileNotJavaScript);
             const inputListOrBundle = (compilerOptions.outFile || compilerOptions.out) ? [createBundle(nonJsFiles)] : nonJsFiles;
-            const declarationTransform = transformNodes(resolver, host, compilerOptions, inputListOrBundle, [transformDeclarations], /*allowDtsFiles*/ false);
+            const declarationTransform = transformNodes(resolver, host, compilerOptions, inputListOrBundle, addRange([transformDeclarations], declarationTransformers), /*allowDtsFiles*/ false);

```

Comment:
Might not matter, but `concatenate` might be faster than `addRange`. `addRange` is more useful when you need to mutate the first array.